python/sbp/client/handler.py

```diff
@@ -180,10 +180,15 @@ def add_callback(self, callback, msg_type=None):
     ----------
     callback : fn
       Callback function
-    msg_type : int
+    msg_type : int | list
       Message type to register callback against. Default `None` means global callback.
+      List type adds the callback to all the message types.
     """
-    self.callbacks[msg_type].add(callback)
+    if not isinstance(msg_type, list):
```

Comment:
This could be generalized to all iterables. The most "pythonic" way to check if something is an iterable is just to assume it is and catch failure:

```python
try:
  for mt in msg_type:
    self.callbacks[mt].add(callback)
catch TypeError:
  self.callbacks[msg_type].add(callback)
```

http://stackoverflow.com/questions/1952464/in-python-how-do-i-determine-if-an-object-is-iterable